cocos/scripting/js-bindings/manual/jsb_global.cpp

```diff
@@ -103,7 +105,59 @@ bool jsb_run_script(const std::string& filePath)
 
         se::ScriptEngine::getInstance()->setFileOperationDelegate(delegate);
     }
+}
 
+bool jsb_enable_debugger(const std::string& debuggerServerAddr, uint32_t port)
+{
+    if (debuggerServerAddr.empty() || port == 0)
+        return false;
+
+    auto se = se::ScriptEngine::getInstance();
+    se->enableDebugger(debuggerServerAddr.c_str(), port);
+
+    // For debugger main loop
+    class SimpleRunLoop
+    {
+    public:
+        void update(float dt)
+        {
+            se::ScriptEngine::getInstance()->mainLoopUpdate();
+        }
+    };
+    static SimpleRunLoop runLoop;
+    Director::getInstance()->getScheduler()->scheduleUpdate(&runLoop, 0, false);
+    return true;
+}
+
+bool jsb_set_extend_property(const char* ns, const char* clsName)
+{
```

Comment:
A wrapper for easily setting xxx.extend = cc.Class.extend.
Don't use se::ScriptEngine::evalString since it will shown in `Browser Debugger` with `(no filename)`. And it's useless.